test/kernel_apps.js

```diff
@@ -132,12 +132,24 @@ contract('Kernel apps', accounts => {
                     await kernel.setApp(APP_BASE_NAMESPACE, appId, appCode1.address)
                 })
 
+                it('fails calling function with isInitialized (if it\'s not)', async () => {
```

Comment:
Useful to note, you can use double quotes `"` to avoid the `\'` :).

Reply:
Haha, yes I know, I was just trying to be consistent with the rest of the file (and unsure of if there is any guideline about the use of single vs double quotes)